documentation/internal/Release.md

```diff
@@ -0,0 +1,15 @@
+# Releasing KubeMarine
+
+If you want to make a new KubeMarine release, you need to do following:
+1. On the target git branch, update KubeMarine version and create tag (replace `X.X.X` with actual version):
+    ```
+    python3 -m pip install bumpver
+    python3 -m bumpver update --set-version=X.X.X
+    ```
+2. Create a new pre-release for your tag on [GitHub Release page](https://github.com/Netcracker/KubeMarine/releases). Do not forget to generate release notes.
```

Comment:
This step is not needed , since tag and pre-release are done by CI

Reply:
Removed